packages/cli/src/commands/election/run.ts

```diff
@@ -1,7 +1,23 @@
+import { ContractKit } from '@celo/contractkit/lib'
 import { cli } from 'cli-ux'
 import { BaseCommand } from '../../base'
 import { validatorTable } from '../validator/list'
 
+async function performElections(kit: ContractKit) {
+  const election = await kit.contracts.getElection()
+  const election3 = await kit._web3Contracts.getElection()
+  try {
+    const signers = await election.electValidatorSigners()
+    return signers
+  } catch (err) {
+    console.warn('Warning: error running actual elections, retrying with minimum validators at 0')
+    const config = await election.getConfig()
+    return election3.methods
```

Comment:
Can we add this to the contractKit wrapper here instead?

If it's easy, we could even consider making the `electValidatorSigners` in the wrapper take optional `min` and `max` arguments. If neither are provided, call `electValidatorSigners`, if at least one is provided, call `electNValidatorSigners`